### ISSUE TITLE
Questions about PDE loss

### ISSUE THREAD

**Issue by mingze-yuan** [jhhuangchloe/DiffusionPDE#5]
Hi author,

Thanks for your great work! 

I have some confusion on the code computing PDE loss, for example, in [1](https://github.com/jhhuangchloe/DiffusionPDE/blob/7c10f54c6a749735a99287fbf8cb4bc2547f5bea/scripts/generate_darcy.py#L24), why is it computed by torch.tensor([[1, 0, -1]] as kernel? I think it should be torch.tensor([[-1, 0, 1]], could you please explain it? Also in the other losses for Burgers, etc. 

Btw, there is a small inconsistency in the dataset generation code with the sampling code or the paper. [2](https://github.com/jhhuangchloe/DiffusionPDE/blob/7c10f54c6a749735a99287fbf8cb4bc2547f5bea/dataset_generation/static/generate_darcy.m#L34)
Here, the lower bound should be 3, instead of 4.

Best,
Mingze

**Comment by jhhuangchloe** [comment]
Hello Mingze,

Thank you for bringing these concerns to our attention! We have just corrected the errors.
